src/cppgc_helpers.h

```diff
@@ -88,8 +99,29 @@ class CppgcMixin : public cppgc::GarbageCollectedMixin {
     visitor->Trace(traced_reference_);
   }
 
+  // This implements CppgcWrapperListNode::Clean and is run for all the
+  // remaining Cppgc wrappers tracked in the Environment during Environment
+  // shutdown. The destruction of the wrappers would happen later, when the
+  // final garbage collection is triggered when CppHeap is torn down as part of
+  // the Isolate teardown. If subclasses of CppgcMixin wish to perform cleanups
+  // that depend on the Environment during destruction, they should implment it
+  // in a CleanEnvResource() override, and then call this->Clean() from their
+  // destructor. Outside of CleanEnvResource(), subclasses should avoid calling
+  // into JavaScript or perform any operation that can trigger garbage
+  // collection during the destruction.
+  void Clean() override {
```

Comment:
Looks like a good pattern of `Cleanable`. Can this be merged into the `Environment::cleanable_queue_`?

https://github.com/nodejs/node/blob/b8f6d84ff5893b1274d658da22bc4067c2fb93fa/src/env.h#L606-L616

Reply:
That was what I did at first :) but then I realized if it's in that queue, to filter out all the wraps it would be an unnecessary hassle. Might as well just keep them in a separate queue and there would be no need for filtering.